components/sensor/atm90e32.rst

```diff
@@ -63,6 +69,8 @@ Configuration variables:
 - **phase_c** (*Optional*): The configuration options for the 3rd phase. Same options as 1st phase.
 - **frequency** (*Optional*): Use the frequenycy value calculated by the meter. All options from
   :ref:`Sensor <config-sensor>`.
+- **peak_current_signed** (*Optional*): Control the peak current output as signed or absolute. All options from
+  :ref:`Sensor <config-sensor>`.
```

Comment:
This doesnt exist in the code PR?

Reply:
Hmm, I will check what happened, it is on one of my local repos, I may have added it to the wrong local branch.

Reply:
Thanks for the eagle eyes on this. Ok I have cherry picked the missing piece from the local copy.